scripts/ruby_common/slack_client.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
```

Comment:
this seems to be a copy from https://github.com/ChainSafe/forest/tree/main/scripts/sync_check/src, should we re-use it?